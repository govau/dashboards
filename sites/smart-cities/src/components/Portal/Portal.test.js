import React from 'react';
import { shallow } from 'enzyme';
import Portal from './Portal';

it('should match Snapshot', () => {
  const component = shallow(
    <Portal>Hi</Portal>
  );

  expect(component.debug()).toMatchSnapshot();
});
