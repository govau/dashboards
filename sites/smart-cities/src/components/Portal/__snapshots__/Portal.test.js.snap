// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should match Snapshot 1`] = `"<undefined />"`;
